src/qt/bitcoin.h

```diff
@@ -112,7 +116,7 @@ public Q_SLOTS:
     void windowShown(BitcoinGUI* window);
 
 private:
-    QThread *coreThread;
+    std::unique_ptr<BitcoinCore> m_executor;
```

Comment:
In commit "qt: Add BitcoinCore::m_thread member" (efacacba98aa6d9d0794f62032eb353ced1e0791)

Not important, but this could use `std::optional` instead of `std::unique_ptr` to avoid a dynamic allocation

Reply:
Thank! [Updated](https://github.com/bitcoin-core/gui/pull/381#issuecomment-880133880).

Reply:
very good